andela/travel_tool_front
# #160714903 View accommodation details as part of  request modal

#### What does this PR do?
This PR enables a requester to view accommodation details as part of the request modal

#### Description of Task to be completed?
Upon login and making a successful request on Travela, when a user wants to view/confirm the details of the request they have made, they should view the accommodation details as well. Similarly, when a manager logins in to approve a user's requests, they should see the accommodation arrangement of the user.

#### How should this be manually tested?

- Clone this repo
- Change into the project directory
- Check out this branch with git checkout ft-view-accomodation-details-160714903
- Start the server with make start
- Navigate to localhost:3000 on your browser
- Open your requests and choose a particular request to view full details
- The accommodation details should now be displayed on the request modal as part of the trip details

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?

[160714903](https://www.pivotaltracker.com/story/show/160714903)
#### Screenshots (if appropriate)

<img width="940" alt="screen shot 2018-10-03 at 10 53 10" src="https://user-images.githubusercontent.com/37342753/46397391-cee46b80-c6fa-11e8-93b6-7a53b7624d4d.png">



#### Questions:
